src/rules/ig.toml

```diff
@@ -0,0 +1,23 @@
+min_trimmed_length = 3
+min_word_count = 3
+max_word_count = 14
+min_characters = 0
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+needs_uppercase_start = false 
+disallowed_symbols = [
```

Comment:
`disallowed_symbols` got replaced by `allowed_symbols` as it better filters out non-useful characters. We are currently in the progress of changing that in the English file as well, you can take a sneak peek here: https://github.com/common-voice/cv-sentence-extractor/pull/159/files#diff-0ba36324ff3a950efd76d64d802802688fdfe6bea026934e7f3cb9cd3140c305R12

Reply:
I'm mostly offline the next few days, maybe @stefangrotz could help out here. However I don't see this as a necessity per se, if the quality is good enough.

Reply:
Hey, what problems do you have with adding these letters @chrisemezue? It should be able to simply add them all. 

For Esperanto I simply collected all letters (uppercase and lowercase) of the alphabet, and it worked, this might be the easiest solution for you:

`allowed_symbols_regex="[AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpRrSsTtUuVvZzĉĈĝĜĥĤĵĴŝŜŭŬ;‚'–\\. \"\\?!«»()]"`

Reply:
I see. If you want to find out which symbols you want to filter, I can recommend using for example the word count file and then use 

`sed 's/\(.\)/\1\n/g' your-text.txt | sort | uniq -ic > characters-count.txt`

This will give you a file where you easily can collect unwanted signs for the filter.

Reply:
thank you for this line of code. very useful.